### PR TITLE
Code for creating sub-directories for app/

### PR DESCRIPTION
**Pull request by ashutoshnarayan**
Python code for creating sub-directories under app/ to maintain handlers, data, utils respectively.

This is for #212 .

### CODE REVIEW

**Comment by gimite** [comment]
Thanks for the pull request! A few comments:

* Sorry forgot to mention, but you don't need to move files already in subdirectories i.e., app/locale/, app/recaptcha/, app/resources/, app/vendors/
* Looks something is wrong with your script that it copies a lot of files under app/xlrd/ e.g., app/app.yaml -> app/xlrd/app.yaml. Can you take a look?
* You also need to update paths in `import` lines for the moved files.
* Looks like your branch has some conflicts with the latest master branch, as in the message above. Maybe you branched from an old repository? Can you pull and merge the latest master branch?

File: app/.cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
[the remark this answers is not in the document]
Comment (ashutoshnarayan) [reply app/.cache/v/cache/lastfailed]: have removed .cache/ directory and added to .gitignore

[the remark this answers is not in the document]
Comment (ashutoshnarayan) [reply app/.cache/v/cache/lastfailed]: Have removed .cache/ directory and added to .gitignore

**Comment by gimite** [comment]
* Can you also move all the remaining app/*.py into app/utils/, except for main.py, model.py? Looks like you have some files remaining like const.py.
* Looks like you have git conflict again. Can you make sure that you merged the latest HEAD of the master branch?
* I guess you are planning to work on this too?:
    > You also need to update paths in `import` lines (and any other references) for the moved files. Note that HANDLER_CLASSES in main.py must be updated as well.

**Comment by ashutoshnarayan** [comment]
> * Do not move app.yaml, cron.yaml, index.yaml. They need to be directly under app/.
Have left .yaml files under app/

> * Looks there is still a conflict. The [Github pull request page](https://github.com/google/personfinder/pull/448) says:
>   > This branch has conflicts that must be resolved

The conflict is coming because the master branch has .py files present while through my code those are moved to different directory ; I also tried by creating a new branch but here too same issue. Can you please check why is this causing issue ?

**Comment by gimite** [comment]
> The conflict is coming because the master branch has .py files present while through my code those are moved to different directory ; I also tried by creating a new branch but here too same issue. Can you please check why is this causing issue ?

As far as I can see in https://github.com/google/personfinder/network , looks like your master branch (ashutoshnarayan:master) forked from our master branch (google:master) two years ago and have never merged google:master since then.

Can you branch from the latest HEAD of google:master and apply changes again? I believe you can do that by:

* Follow [this Github doc](https://help.github.com/articles/configuring-a-remote-for-a-fork/) if you haven't.
* Run these commands:
    ```
    $ git fetch upstream
    $ git checkout -b new-branch upstream/master
    ... apply your changes again...
    $ git push origin new-branch
    ```
* Send a pull request with new-branch again. (`new-branch` can be an arbitrary branch name).

I haven't tried it myself, though. Also refer to [this doc](https://help.github.com/articles/syncing-a-fork/) for details, though you probably don't want to follow the doc as is, which merges our 2 year changes with your change and causes a lot of conflicts.

**Comment by ashutoshnarayan** [comment]
> Can you branch from the latest HEAD of google:master and apply changes again? I believe you can do that by:

Have applied the changes to a new branch and raised a PR #450 against it.

**Comment by gimite** [comment]
Thanks! Closing this one.

**Pull request closed without merging**